storage/bucket.go

```diff
@@ -82,8 +82,13 @@ func (c *Client) Bucket(name string) *BucketHandle {
 // Create creates the Bucket in the project.
 // If attrs is nil the API defaults will be used.
 func (b *BucketHandle) Create(ctx context.Context, projectID string, attrs *BucketAttrs) (err error) {
-	ctx = trace.StartSpan(ctx, "cloud.google.com/go/storage.Bucket.Create")
-	defer func() { trace.EndSpan(ctx, err) }()
+	if isOTelTracingDevEnabled() {
+		ctx, _ = startSpan(ctx, "cloud.google.com/go/storage.Bucket.Create")
```

Comment:
Since `startSpan` is completely internal to Storage now, can we factor out `cloud.google.com/go/storage`?

Reply:
Absolutely, I've changed these to `storage.Bucket.Create` so we're still following the semantic conventions of $package.$service.$method

Reply:
Oh I meant, just to move `cloud.google.com/go/` to the `startSpan` method so that we don't have to repeat it in the name of each span (and to help with consistency).